ostypes_win.jsm

```diff
@@ -34,6 +34,7 @@ var winTypes = function() {
 	this.BYTE = ctypes.unsigned_char;
 	this.CHAR = ctypes.char;
 	this.DWORD = ctypes.unsigned_long; // IntSafe.h defines it as: // typedef unsigned long DWORD; // so maybe can change this to ctypes.unsigned_long // i was always using `ctypes.uint32_t`
+	this.LPDWORD = this.DWORD.ptr;
```

Comment:
Duplicate. Already defined in the `ADVANCED TYPES` section. As `LPDWORD` depends on `DWORD`.